app/controllers/posts_controller.rb

```diff
@@ -27,27 +27,25 @@ def update
       render 'edit'
     end
   end
-# # search
-#   def search
-#     if query.present?
-#       Post.index
-#     else
-#       scoped
-#     end
-#   end
-
-#   def search
-#     Post.search params[:query]
-#   end
```

Comment:
Before you start fixing any bugs or make any changes, please keep your master up to date with main master repo.